entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityOperationHandler.kt

```diff
@@ -50,8 +47,8 @@ class EntityOperationHandler(
             }
         )
 
-        authorizationService.createAdminLinks(batchOperationResult.success, userId)
-        ngsiLdEntities.filter { it.id in batchOperationResult.success }
+        authorizationService.createAdminLinks(batchOperationResult.success.map { it.entityId }, userId)
```

Comment:
you're doing this `batchOperationResult.success.map { it.entityId }` in a lot of places, I think you could make it a function of the `BatchOperationResult` data class.